model/articles.model.js

```diff
@@ -68,16 +68,25 @@ function updateArticleById(article_id, newVote) {
     )
     .then((result) => {
       if (result.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "Failed to update becasue the article does not exist" });
+        return Promise.reject({
+          status: 404,
+          msg: "Failed to update becasue the article does not exist",
+        });
       }
       return result.rows[0];
     });
 }
 
+function removeCommentById(comment_id) {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1`,[comment_id])
+}
```

Comment:
You'll want a custom error here for your 404 non existent ID test 💭  
💡 checking the rowCount would be helpful for this